crates/wasmtime/src/runtime/vm.rs

```diff
@@ -183,7 +183,7 @@ cfg_if::cfg_if! {
 /// APIs using `Store<T>` are correctly inferring send/sync on the returned
 /// values (e.g. futures) and that internally in the runtime we aren't doing
 /// anything "weird" with threads for example.
-pub unsafe trait VMStore {
+pub unsafe trait VMStore: 'static {
```

Comment:
No big deal, but curious why you decided to add this bound again.  Was it unavoidable given the use in TLS?